Fitbit.Portable.Tests/SleepTests.cs

```diff
@@ -14,6 +14,8 @@ namespace Fitbit.Portable.Tests
     [TestFixture]
     public class SleepTests
     {
+        #region Success Test
```

Comment:
@aarondcoleman are you a fan of using regions to break apart sections of code or would you rather not have these? i included them  since the file was starting to get pretty big.